pipeline/metadata/schema.py

```diff
@@ -250,35 +250,35 @@ def flatten_for_bigquery_satellite(row: SatelliteRow) -> Dict[str, Any]:
   """Convert a structured satellite dataclass into a flat dict."""
   flat: Dict[str, Any] = {
       'domain': row.domain,
-      'category': row.category,
-      'ip': row.ip,
+      'domain_category': row.category,
+      'is_control_domain': row.is_control,
       'date': row.date,
       'start_time': row.start_time,
       'end_time': row.end_time,
-      'error': row.error,
-      'anomaly': row.anomaly,
+      'resolver_ip': row.ip,
+      'resolver_name': row.ip_metadata.name,
+      'is_control_resolver': row.is_control_ip,
+      'resolver_netblock': row.ip_metadata.netblock,
+      'resolver_asn': row.ip_metadata.asn,
+      'resolver_as_name': row.ip_metadata.as_name,
+      'resolver_as_full_name': row.ip_metadata.as_full_name,
+      'resolver_as_class': row.ip_metadata.as_class,
+      'resolver_country': row.ip_metadata.country,
+      'resolver_organization': row.ip_metadata.organization,
+      'received_error': row.error,
+      'received_rcode': row.rcode,
+      'answers': [],
       'success': row.success,
-      'is_control': row.is_control,
-      'controls_failed': row.controls_failed,
-      'measurement_id': row.measurement_id,
-      'source': row.source,
-      'is_control_ip': row.is_control_ip,
-      'rcode': row.rcode,
+      'anomaly': row.anomaly,
+      'domain_controls_failed': row.controls_failed,
       'average_confidence': row.average_confidence,
       'untagged_controls': row.untagged_controls,
       'untagged_response': row.untagged_response,
       'excluded': row.excluded,
       'exclude_reason': row.exclude_reason,
       'has_type_a': row.has_type_a,
```

Comment:
I think we can remove the `has_type_a` field (indicating if there's at least one `rcode=0` and `ip != null` response for a DNS query), since we already have a `rcode` field